### PR TITLE
add pushover, webhook receivers, using namespaced config

### PR DESCRIPTION
**Pull request by sed-i**

### CODE REVIEW

File: src/config.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
+import abc
+from typing import Optional
+
+
+class ConfigBase(abc.ABC):
```
Comment (balbirthomas) [inline src/config.py:5]: docstrings ? : What is this class for ? How is it intended to be used ?

File: src/config.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    def from_dict(data: dict) -> Optional[dict]:
+        ...
+
+    @staticmethod
+    @abc.abstractmethod
+    def is_valid(data: dict) -> bool:
+        ...
+
+
+class ReceiverBase(ConfigBase):
```
Comment (balbirthomas) [inline src/config.py:18]: docstring ?

**Review by balbirthomas — APPROVED** [review]
Approving. But docstrings would be very useful for each of your classes explaining what they are meant for and how they are intended to be used.

File: src/config.py (hunk 1 above)
Comment (sed-i) [reply src/config.py:5]: Added

File: src/config.py (hunk 2 above)
Comment (sed-i) [reply src/config.py:26]: Added

**Review by Abuelodelanada — APPROVED** [review]
LGTM

**Pull request merged**